crates/trycp_server/src/configure_player.rs

```diff
@@ -57,88 +54,28 @@ pub(crate) fn configure_player(
         players_guard.insert(
             id.clone(),
             Player {
-                admin_port: port,
+                admin_port,
                 processes: Mutex::default(),
             },
         );
     }
 
-    let lair_stderr_log_path = player_dir.join(LAIR_STDERR_LOG_FILENAME);
-    let mut lair = Command::new("lair-keystore")
-        .current_dir(&player_dir)
-        .env("RUST_BACKTRACE", "full")
-        .args(["init", "--piped"])
-        .stdin(Stdio::piped())
-        .stdout(Stdio::piped())
-        .stderr(
-            std::fs::OpenOptions::new()
-                .append(true)
-                .create(true)
-                .open(&lair_stderr_log_path)
-                .context(CreateLairConfig {
-                    path: lair_stderr_log_path.clone(),
-                })?,
-        )
-        .spawn()
-        .context(CreateLairConfig {
-            path: lair_stderr_log_path.clone(),
-        })?;
-    if let Some(mut lair_stdin) = lair.stdin.take() {
-        lair_stdin
-            .write_all(LAIR_PASSPHRASE.as_bytes())
-            .context(CreateLairConfig {
-                path: lair_stderr_log_path.clone(),
-            })?;
-    }
-    lair.wait().context(CreateLairConfig {
-        path: lair_stderr_log_path.clone(),
-    })?;
-
-    // get connection url from lair config
-    let lair_url = Command::new("lair-keystore")
-        .current_dir(&player_dir)
-        .env("RUST_BACKTRACE", "full")
-        .arg("url")
-        .stdout(Stdio::piped())
-        .stderr(
-            std::fs::OpenOptions::new()
-                .append(true)
-                .create(true)
-                .open(&lair_stderr_log_path)
-                .context(CreateLairConfig {
-                    path: lair_stderr_log_path.clone(),
-                })?,
-        )
-        .spawn()
-        .context(CreateLairConfig {
-            path: lair_stderr_log_path.clone(),
-        })?;
-    let connection_url_output = lair_url.wait_with_output().context(CreateLairConfig {
-        path: lair_stderr_log_path.clone(),
-    })?;
-    let connection_url = std::str::from_utf8(connection_url_output.stdout.as_slice()).unwrap();
-
-    let mut config_file = std::fs::File::create(&config_path).with_context(|| CreateConfig {
-        path: config_path.clone(),
-    })?;
-
-    writeln!(
-        config_file,
+    let config = format!(
         "\
 ---
 data_root_path: environment
 keystore:
-    type: lair_server
-    connection_url: {}
+    type: lair_server_in_proc
```

Comment:
```suggestion
  type: lair_server_in_proc
```

If we're switching to two space indents as below...